README.rst

```diff
@@ -768,7 +766,7 @@ Technical outline of how cwltool works internally, for maintainers.
       using containers) or symlinks (if not).  This staging step enables files
       to be logically rearranged or renamed independent of their source layout.
    #. The ``run()`` method of CommandLineJob executes the command line tool or
-      Docker container, waits for it to complete, collects output, and makes
+      Docker container waits for it to complete, collects output, and makes
```

Comment:
Better to put in-line numbering to clarify this sentence

Reply:
I reverted the change.